corehq/apps/app_manager/tests/test_child_module.py

```diff
@@ -68,7 +68,7 @@ def test_workflow_with_put_in_root(self):
             </text>
             <command id="m0-f0"/>
           </menu>
-          <menu id="m0">
```

Comment:
This doesn't actually work. If you change this menu ID then it will appear as a separate menu instead of including the forms in the parent module. I've updated the child module docs to include a section on the menu structure: https://github.com/dimagi/commcare-hq/pull/9923